charts/jaeger-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.21.1
```

Comment:
I think it's more of a minor change `2.21.2`

Reply:
fixed to minor version